### PR TITLE
GitHub sync

### PR DESCRIPTION
**Pull request by nshahan**

### CODE REVIEW

File: angular_components/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,115 @@
+## 0.14.0-alpha+1
+### Component Updates
+
+#### Application Layout
+* Add `canClose` input the temporary drawer to prevent the drawer from toggling.
```
Comment (kwalrath) [inline angular_components/CHANGELOG.md]: I don't understand this one. Is a word missing?

File: angular_components/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,115 @@
...
+* Add input to toggle selection of non-leaf nodes.
+
+### Other Updates
+
+#### Miscellaneous
+* Remove `mat-icon-image()` in favor of `inline-image()`.
+* Add `subtract()` to Date model.
+* Migrate event handlers with multiple statements to component methods.
+* Fix violations of the `strict-raw-types` analysis option.
+* Fix violations of `prefer_initializing_formals`, `prefer-collection-literals`,
```
Comment (kwalrath) [inline angular_components/CHANGELOG.md]: `-` -> `_`
(in the lint names)

File: angular_components/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,115 @@
...
+#### Material Tree
+* Add input to toggle selection of non-leaf nodes.
+
+### Other Updates
+
+#### Miscellaneous
+* Remove `mat-icon-image()` in favor of `inline-image()`.
+* Add `subtract()` to Date model.
+* Migrate event handlers with multiple statements to component methods.
+* Fix violations of the `strict-raw-types` analysis option.
```
Comment (kwalrath) [inline angular_components/CHANGELOG.md]: `strict-raw-types` analysis option -> `--strict-raw-types` analyzer option?
(at first I was looking for "strict" in the linter options)

**Comment by nshahan** [comment]
Thanks! I updated the CHANGELOG so they will look cleaner in the future, but I don't think the errors are gross enough to go through the dance of syncing again.

File: angular_components/CHANGELOG.md (hunk 3 above)
Comment (nshahan) [reply angular_components/CHANGELOG.md]: Done.

**Pull request merged**